bench/f32-rsum.cc

```diff
@@ -269,6 +269,36 @@
 #endif  // XNN_ENABLE_RISCV_VECTOR && XNN_ARCH_RISCV
 
 
+#if XNN_ENABLE_RISCV_VECTOR && XNN_ARCH_RISCV
+  BENCHMARK_CAPTURE(f32_rsum, rvv_u2v,
+                    xnn_f32_rsum_ukernel__rvv_u2v,
+                    xnn_init_f32_scale_scalar_params,
+                    benchmark::utils::CheckRVV)
+    ->Apply(BenchmarkRSUM)
+    ->UseRealTime();
+#endif  // XNN_ENABLE_RISCV_VECTOR && XNN_ARCH_RISCV
```

Comment:
Do you need the endifs here? The condition for benchmarking the 2, 4 and 8 case are the same, so just check once

Reply:
The bench/f32-rsum.cc file is auto-updated using tools/generate-rdsum-benchmark.py, this has generated individual endifs for each case.